client/src/templates/Introduction/components/ClaimCertSteps.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+import { withTranslation, useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import GreenPass from '../../../assets/icons/GreenPass';
+import GreenNotCompleted from '../../../assets/icons/GreenNotCompleted';
+import { userSelector } from '../../../redux';
+import { User } from '../../../redux/propTypes';
+
+const mapIconStyle = { height: '15px', marginRight: '10px', width: '15px' };
+const renderCheckMark = isCompleted => {
+  return isCompleted ? (
+    <GreenPass style={mapIconStyle} />
+  ) : (
+    <GreenNotCompleted style={mapIconStyle} />
+  );
+};
+
+const propTypes = {
+  i18nCertText: PropTypes.string,
+  superBlock: PropTypes.string,
+  user: User
+};
+
+const mapStateToProps = state => {
+  return createSelector(userSelector, user => ({
+    user
+  }))(state);
+};
+
+const ClaimCertSteps = ({ i18nCertText, superBlock, user }) => {
+  const { t } = useTranslation();
+
+  const settingsLink = '/settings#privacy-settings';
+  const certClaimLink = `/settings#settings-${superBlock}`;
```

Comment:
The `certSlug` and the `superBlock` are not the same. So, this does not work for `-v7` certifications